src/composables/useFlawsFetching.ts

```diff
@@ -14,7 +14,15 @@ export function useFlawsFetching() {
     const requestParams: Record<string, any> = {};
     for (const key in params) {
       if (params[key] === '' && allowedEmptyFieldMapping[key]) {
-        requestParams[allowedEmptyFieldMapping[key]] = 1;
+        const fields = allowedEmptyFieldMapping[key];
+        // CVSS3 score field purpose now
+        if (Array.isArray(fields)) {
+          fields.forEach((field) => {
+            requestParams[field] = 1;
+          });
```

Comment:
Does this mean that if using `cvss_scores__score`, it will filter by emptiness on all CVSS versions and issuers?

Reply:
Yeah, Like this.
![image](https://github.com/RedHatProductSecurity/osim/assets/156366006/8c553bd6-d042-457c-a3be-161f33916e0e)